### PR TITLE
Add loglevel settings, default rlp_03 logging to info, some minor pro…

### PR DESCRIPTION
**Pull request by StrongestNumber9**
…perties cleanup

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -22,9 +22,7 @@ Custom server keystore can be supplied with "-DtlsKeystore=path/to/file.jks".
 
 Custom server keystore password can be supplied with "-DtlsKeystorePassword=MyCustomPassword".
 
-Debug output
-
-See src/main/resources for the log4j2.xml
+Logging level output can be managed with "-Dloglevel=INFO".
```
Comment (MoonBow-1) [inline README.adoc:25]: Link to possible options?

**Review by MoonBow-1 — COMMENTED** [review]
Could use a link to all possible log levels, but not mandatory.
LGTM

**Review by MoonBow-1 — APPROVED** [review]

File: README.adoc (hunk 1 above)
Comment (StrongestNumber9) [reply README.adoc]: Added a link

Comment (StrongestNumber9) [reply README.adoc]: Added a clarification to mention it only affects the internal component logging and not the severity of the event itself

**Review by MoonBow-1 — APPROVED** [review]
LGTM with the log level documentation link

**Pull request merged**